src/python/grapl-plugin-sdk/grapl_plugin_sdk/analyzer/service_impl.py

```diff
@@ -69,6 +69,7 @@ class AnalyzerServiceImpl:
 
     def __post_init__(self) -> None:
         self._graph_query = self._analyzer.query().into_graph_query()
+        LOGGER.info("Graph query", graph_query=str(self._graph_query))
```

Comment:
This happens once at the top of the analyzer and helped me debug some stuff, so, keeping it here